src/account/accountReducers.js

```diff
@@ -79,9 +79,8 @@ export default (state = initialState, action) => {
       ];
     }
     case LOGOUT: {
-      const newState = state.slice();
-      newState[0].apiKey = '';
-      return newState;
+      // Remove the active account
```

Comment:
I think it's confusing if the account remains in the list when the user logs out. My expectation (as a user) would be that logging out removes the account entirely from the app.

This also fixes an issue where `AuthScreen` (or `PasswordAuthScreen` previously) appears but there's no auth backend data when rehydrating state. That's why I'm including it in this PR :)